techniques/aria/ARIA11.html

```diff
@@ -110,11 +112,11 @@ <h3>Search form</h3>
                   <a href="https://www.w3.org/TR/wai-aria/#states_and_properties">Accessible Rich Internet Applications (WAI-ARIA) 1.0, Supported States and Properties</a>
                </li>
             <li>
-                  <a href="https://www.paciellogroup.com/blog/2013/07/enabling-landmark-based-keyboard-navigation-in-firefox/">Enabling landmark-based keyboard navigation in Firefox</a>
+                  <a href="https://developer.paciellogroup.com/blog/2017/05/improving-access-to-landmark-navigation/">Enabling landmark-based keyboard navigation in Firefox</a>
```

Comment:
new URL (though this old one redirects) https://www.tpgi.com/improving-access-to-landmark-navigation/

Reply:
```suggestion
                  <a href="https://www.tpgi.com/improving-access-to-landmark-navigation/">Enabling landmark-based keyboard navigation in Firefox</a>
```